components/private/KeyboardWatcher/BUILD

```diff
@@ -21,4 +24,34 @@ mdc_public_objc_library(
     deps = [
         "//components/private/Application",
     ],
+    sdk_frameworks = [
+        "CoreGraphics",
```

Comment:
+ "UIKit" ?

Reply:
Bazel didn't complain, so I didn't add it?

Reply:
Looks like it uses UIKit, and other BUILDs have it there so I would maybe be on the safe side and add it, but upto you, my knowledge of Bazel is limited at this point

Reply:
Added the dependency.  Generally, I follow a policy of "if the tool doesn't need it, don't use it. "

Reply:
Taken from https://docs.bazel.build/versions/master/be/objective-c.html

![screen shot 2017-12-20 at 12 02 17 pm](https://user-images.githubusercontent.com/4066863/34218948-b0cbb2b6-e57d-11e7-95de-6b5070d6dec6.png)

I guess it's a NOOP to add it or not. For consistency sake I would add it, if we want to make it cleaner, we should then remove it from all other BUILDs